pyEpiabm/pyEpiabm/tests/test_unit/test_core/test_household.py

```diff
@@ -36,6 +36,19 @@ def test_location_type(self):
         self.assertRaises(ValueError, pe.Household,
                           self.microcell, ('1', 1))
 
+    def test_add_susceptible_person(self):
+        subject = pe.Household(self.microcell, (1, 1))
+        self.assertEqual(len(subject.susceptible_persons), 0)
+        subject.add_susceptible_person(self.person)
+        self.assertEqual(len(subject.susceptible_persons), 1)
+
+    def test_remove_susceptible_person(self):
+        subject = pe.Household(self.microcell, (1, 1))
+        subject.susceptible_persons.append(self.person)
+        self.assertEqual(len(subject.susceptible_persons), 1)
+        subject.remove_susceptible_person(self.person)
+        self.assertEqual(len(subject.susceptible_persons), 0)
+
```

Comment:
It's great to checking the length for susceptible_pesons. It's better to also check if self.person.household equals subject (i.e. that's what you code in L85, L98)